web-console/package.json

```diff
@@ -23,7 +23,7 @@
     "format": "prettier --write \"{src,tests}/**/*.{js,jsx,ts,tsx}\"",
     "format:check": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\"",
     "generate-openapi": "openapi --input ../openapi.json --output ./src/lib/services/manager",
-    "build-generate-openapi": "cargo run --bin pipeline-manager -- --dump-openapi && openapi --input ./openapi.json --output ./src/lib/services/manager && rm ./openapi.json",
+    "build-openapi": "cargo run --bin pipeline-manager -- --dump-openapi",
```

Comment:
so the openapi.json file will be there or not?

Reply:
I am leaving this command as a dev convenience only; the openapi.json is still supposed to be generated in the same PR where pipeline_manager API updated.